guardian/typerighter
# Add Tags table

## What does this change?

This PR adds: 
- a Tags page at the `/tags` path
- a table of tags, listing how many rules they are associated with
- a new method to count the rules associated with a tag
- a 'create tag' form
- inline-editing in the table for tag names
- a 'delete tag' action on the tag table, with a confirmation action

## How to test

Run the application locally according to the instructions in the readme. Enable the Tags page feature switch in teh dropdown in the top right of the rule manager.

If you have Typerighter permissions, can you do all of the following:
- List all tags, and see the number of rules they're associated with?
- Delete a tag?
- Edit a tag name?

If you lack edit permissions, are you prevented from creating, editing, or deleting rules?

## Images

![Kapture 2023-07-12 at 16 27 47](https://github.com/guardian/typerighter/assets/34686302/cfbc039b-62bc-477e-a29c-d9790ff2d6bf)
